host/scripts/new_instance.sh

```diff
@@ -2,12 +2,21 @@
 echo "starting $1 for $2 on localhost:$3"
 mkdir -p containers
 rm -r containers/$1
-cp -r sub containers/$1
+cp -r template containers/$1
 cd containers/$1
 rm .env
 rm -r venv
 touch .env
 echo "ORG_NAME=$2" >> .env
 echo "COMPOSE_PROJECT_NAME=$1" >> .env
 echo "EXTERNAL_PORT=$3" >> .env
+
+# unfinished config stuff
+echo "MAIL_USERNAME=hack4impact-sendgrid" >> .env
+echo "MAIL_PASSWORD=$MAIL_PASSWORD" >> .env
```

Comment:
won't work on your system since i set $MAIL_PASSWORD in my venv to our sendgrid password